experimental/packages/opentelemetry-instrumentation/test/node/EsmInstrumentation.test.mjs

```diff
@@ -26,14 +26,21 @@ class TestInstrumentationWrapFn extends InstrumentationBase {
     super('test-esm-instrumentation', '0.0.1', config);
   }
   init() {
-    console.log('test-esm-instrumentation initialized!');
```

Comment:
I see that you removed this log, but all other tests below have a similar log. Would you mind keeping it consistent? So keeping this one or removing for all others.